scripts/api_master.py

```diff
@@ -1964,6 +1964,13 @@
                         "keras.utils.Config",
                     ],
                 },
+                {
```

Comment:
can you also add the reference to - https://github.com/keras-team/keras/blob/e61e83df13a43301ff132a62ed998d534ee49089/keras/src/backend/common/remat.py#L137